includes/admin/class-give-settings.php

```diff
@@ -298,7 +298,7 @@ public function give_settings( $active_tab ) {
 						),
 						array(
 							'name'            => __( 'Number of Decimals', 'give' ),
-							'desc'            => __( 'This sets the number of decimal points shown in displayed prices.', 'give' ),
+							'desc'            => __( 'The number of decimal points in displayed prices.', 'give' ),

```

Comment:
Correction: "The number of decimal points displayed in amounts."  -- "Prices" indicates products. But it's a Donation "amount".